src/executables/util_cs.c

```diff
@@ -3283,36 +3281,45 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
 	  if (HA_DISABLED ())
 	    {
+	      check_applied_info = false;
 	      PRINT_AND_LOG_ERR_MSG (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_APPLYINFO,
 						     APPLYINFO_MSG_NOT_HA_MODE));
 	      goto check_applied_info_end;
 	    }
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
-	  (void) db_shutdown ();
-	}
-      else if (check_copied_info)
-	{
-	  memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
-	  strcpy (local_database_name, database_name);
-	  strcat (local_database_name, "@localhost");
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
+	  error = la_get_applied_log_info (database_name, log_path, check_replica_info, verbose, &applied_final_lsa);
+	  if (error != NO_ERROR)
+	    {
+	      check_applied_info = false;
+	      error = NO_ERROR;
+	    }
```

Comment:
```suggestion
	  if (la_get_applied_log_info (database_name, log_path, check_replica_info, verbose, &applied_final_lsa) != NO_ERROR)
	    {
	      check_applied_info = false;
	    }
```

Reply:
error = func();
if(error != NO_ERROR)
This format is planned for the code consistency.

Reply:
even though `error` is not used?

Reply:
OK. I'll follow.